utils/average_checkpoints.py

```diff
@@ -115,7 +121,13 @@ def get_parser():
         default="",
         type=str,
         nargs="?",
-        choices=["acc", "bleu", "cer_ctc", "loss", "perplexity"],
+        choices=["acc", "bleu", "cer", "cer_ctc", "loss", "perplexity"],
+    )
+    parser.add_argument(
```

Comment:
What are the intentions of this option?

Reply:
@sw005320 Sorry. In the initial experiments, I found `cer_ctc` option gave better performance on this corpus. However, it turned out that `acc` option is better than it after parameter tuning. Although `cet_ctc` does not give the best performance, I just want to keep it as an option.

Reply:
OK!